### PR TITLE
Update readme to add more info

### PR DESCRIPTION
**Pull request by adrn**
This fills in some of the currently "TBD" sections!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,9 @@ Astropy Paper III
...
-astropy principles not mentioned in the first paper, the core v2.0
-package, and infrastructure in the astropy project to support development
-in python.
-
-The draft of the paper can be found [here](https://github.com/astropy/astropy-v5.0-paper/blob/master-pdf/main.pdf). -->
-
-TBD
+Our goal is to produce a brief and informative description of major updates
+about the Astropy Project, drawing on relevant changes or news since 2018
+(v2.0).
```
Comment (pllim) [inline README.md:12]: I am guessing you mean the paper version and not `astropy` version?
```suggestion
(paper v2.0).
```

**Review by pllim — COMMENTED** [review]
Just a minor comment. LGTM otherwise.

File: README.md (hunk 1 above)
Comment (nmearl) [reply README.md:12]: I think he is referring to the astropy version. This paper is supposed to correspond to v5.0.

**Review by nmearl — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adrn) [reply README.md:12]: Yea, that's right. Maybe we should just remove the "(v2.0)" entirely?
```suggestion
```

Comment (adrn) [reply README.md]: I pushed a separate fix to remove ambiguity!

**Comment by adrn** [comment]
Thanks @pllim and @nmearl !

**Pull request merged**